picard/ui/mainwindow.py

```diff
@@ -26,6 +26,7 @@
     QtCore,
     QtGui,
     QtWidgets,
+    QtMultimedia
```

Comment:
Please keep order and add a comma (read https://github.com/metabrainz/picard/blob/master/CONTRIBUTING.md#coding-style about `isort`)